crates/biome_cli/src/cli_options.rs

```diff
@@ -66,6 +67,15 @@ pub struct CliOptions {
         display_fallback
     )]
     pub log_kind: LoggingKind,
+
+    #[bpaf(
+        long("diagnostic-level"),
+        argument("info|warn|error"),
+        fallback(Severity::default()),
+        display_fallback
+    )]
+    /// The level of diagnostics to show. In order, from the most verbose to the least verbose: info, warn, error.
```

Comment:
```suggestion
    /// The level of diagnostics to show. In order, from the lowest to the most important: info, warn, error. Passing `--diagnostic-level=error` will cause Biome to print only diagnostics that contain only errors.
```
I offer a different explanation because diagnostics aren't about "verbosity" but "how important is a message".